### PR TITLE
Client83 - Loopback Connect

### PR DESCRIPTION
**Pull request by totojab**

### CODE REVIEW

**Comment by totojab** [comment]
now needs to implement "scrapePair request" in order to get necessary additionnal information when displaying pair detail

File: src/app/pair-detail/pair-detail.component.ts [hunk 1]
```diff
@@ -39,8 +39,8 @@ export class PairDetailComponent implements OnInit, OnDestroy {
   @HostBinding('@routeAnimation') get routeAnimation() {
     return true
   }
-  sneaker: Sneaker = {
-    id: 0, name: '', subName: '', size: '', price: '', brand: '', reviews: [], img: '', images: [''],
+  sneaker: Sneaker = { // TEMP should call base service
```
Comment (Elrendio) [inline src/app/pair-detail/pair-detail.component.ts]: BaseService is already up, switch is possible

File: src/app/services/mock-filter.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Color } from '../shared/base-class'
+import { Size } from '../shared/base-class'
+import { Price } from '../shared/base-class'
+import { PriceRange } from '../shared/base-class'
+import { SneakerFilter } from '../shared/base-class'
+
```
Comment (Elrendio) [inline src/app/services/mock-filter.ts]: You could use only one import

File: src/app/services/sneaker.service.ts [hunk 3]
```diff
@@ -1,21 +1,33 @@
 import { Injectable } from '@angular/core'
+import { Http, Response } from '@angular/http'
+
 import { Sneaker } from '../shared/sneaker'
 import { SneakerFilter } from '../shared/sneaker-filter'
-import { Brand, PriceRange, Size } from '../shared/base-class'
+// import { Brand, PriceRange, Size } from '../shared/base-class'
```
Comment (Elrendio) [inline src/app/services/sneaker.service.ts]: Better remove line then comment

File: src/app/services/sneaker.service.ts [hunk 4]
```diff
@@ -1,21 +1,33 @@
...
-  getSneakers(filters: SneakerFilter): Promise<Sneaker[]> {
-    return Promise.resolve(SNEAKERS.filter(el => {
-      return (!filters.size || correctSize(el.size, filters.size)) &&
-        (!filters.priceRange || inPriceRange(el.price, filters.priceRange)) &&
-        (!filters.brands || inBrands(el.brand, filters.brands))
-    }))
+  private sneakerUrl = 'http://0.0.0.0:3002/api/EverysizePairs/scrapePage'
+  constructor(private http: Http) { }
+
+  // getSneakers(filters: SneakerFilter): Promise<Sneaker[]> {
```
Comment (Elrendio) [inline src/app/services/sneaker.service.ts]: Better delete this function rather then commetn

File: src/app/services/sneaker.service.ts [hunk 5]
```diff
@@ -25,16 +37,16 @@ export class SneakerService {
...
-  return parseInt(ePrice) >= fPriceRange.min.value && parseInt(ePrice) <= fPriceRange.max.value
-}
-function inBrands(eBrand: string, fBrands: Array<Brand>): boolean {
-  let ret_val = false
-  fBrands.forEach(brand => {
-    ret_val = brand === eBrand || ret_val
-  })
-  return ret_val
-}
+// function correctSize(eSize: string, fSize: Size): boolean {
```
Comment (Elrendio) [inline src/app/services/sneaker.service.ts]: Same comment as above

File: src/app/shortlist/favorites/sneaker/sneaker.component.ts [hunk 6]
```diff
@@ -16,10 +16,9 @@ export class SneakerComponent implements OnInit {
   swiperConfig = {
     speed: 400,
     spaceBetween: 20,
-    loop: true,
-    zoom: true,
-    zoomMax: 2,
-    zoomMin: 1,
+    // zoom: true,
```
Comment (Elrendio) [inline src/app/shortlist/favorites/sneaker/sneaker.component.ts]: REmove comments

**Pull request closed without merging**